### PR TITLE
[py] additional type hints in wait.py & command.py

### PR DESCRIPTION
**Pull request by symonk**
Some minor additional type hints for `command.py` & `wait.py`.  Need to figure out the `driver` type hint; but that'll be a decent win throughout once thats sorted in a follow up.

### CODE REVIEW

File: py/selenium/webdriver/support/wait.py [hunk 1]
```diff
@@ -16,15 +16,17 @@
...
 import time
+import typing
+
 from selenium.common.exceptions import NoSuchElementException, InvalidSelectorException
 from selenium.common.exceptions import TimeoutException
 
-POLL_FREQUENCY = 0.5  # How long to sleep in between calls to the method
-IGNORED_EXCEPTIONS = (NoSuchElementException,)  # exceptions ignored during calls to the method
+POLL_FREQUENCY: float = 0.5  # How long to sleep in between calls to the method
+IGNORED_EXCEPTIONS: typing.Iterable[typing.Type[Exception]] = (NoSuchElementException,)  # default to be ignored.
```
Comment (symonk) [inline py/selenium/webdriver/support/wait.py:25]: this is technically a `typing.Tuple`; will change; its the more `ignored_exceptions` in the wait initialiser that would be of this type

File: py/setup.cfg [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [flake8]
-exclude = .tox,docs/source/conf.py
+exclude = .tox,docs/source/conf.py,*venv
```
Comment (symonk) [inline py/setup.cfg]: pretty common to have a venv (I know we work largely with bazel) but this is something that gets in my way a fair bit, I don't think it hurts to have it excluded in here

File: py/selenium/webdriver/support/wait.py [hunk 3]
```diff
@@ -16,15 +16,18 @@
...
 
-POLL_FREQUENCY = 0.5  # How long to sleep in between calls to the method
-IGNORED_EXCEPTIONS = (NoSuchElementException,)  # exceptions ignored during calls to the method
+POLL_FREQUENCY: float = 0.5  # How long to sleep in between calls to the method
+IGNORED_EXCEPTIONS: typing.Tuple[typing.Type[Exception]] = (NoSuchElementException,)  # default to be ignored.
 
 
 class WebDriverWait:
-    def __init__(self, driver, timeout, poll_frequency=POLL_FREQUENCY, ignored_exceptions=None):
+    def __init__(self, driver, timeout: float, poll_frequency: float = POLL_FREQUENCY, ignored_exceptions: typing.Optional[WaitExcTypes] = None):
```
Comment (symonk) [inline py/selenium/webdriver/support/wait.py:30]: on my prios to get black into `tox` to have the line length problems automatically handled etc

**Review by AutomatedTester — APPROVED** [review]

**Comment by AutomatedTester** [comment]
Can you fix the tox stuff and then it's good to go

**Pull request merged**